docs/services/aws-services-guide.md

```diff
@@ -34,154 +29,25 @@ local machine.
-```
-
-#### Create an IAM Role for the EKS cluster
-
-For the EKS cluster to be able to access different resources, it needs to be
-given the required permissions through an **IAM Role**. In this section we
-create a role and assign the required policies. Later we will make EKS to assume
-this role.
-
-```bash
-EKS_ROLE_NAME=crossplane-example-eks-role
-
-# the policy that determines what principal can assume this role
-ASSUME_POLICY='{
-  "Version": "2012-10-17",
-  "Statement": {
-    "Effect": "Allow",
-    "Principal": {
-      "Service": "eks.amazonaws.com"
-    },
-    "Action": "sts:AssumeRole"
-  }
-}'
-
-# Create a Role that can be assumed by EKS
-aws iam create-role \
-  --role-name "${EKS_ROLE_NAME}" \
-  --region "${REGION}" \
-  --assume-role-policy-document "${ASSUME_POLICY}"
-```
-
-The output should be the created role in JSON. Next, you'll attach the required
-policies to this role:
-
-```bash
-# attach the required policies
-aws iam attach-role-policy --role-name "${EKS_ROLE_NAME}" --policy-arn arn:aws:iam::aws:policy/AmazonEKSClusterPolicy > /dev/null
-aws iam attach-role-policy --role-name "${EKS_ROLE_NAME}" --policy-arn arn:aws:iam::aws:policy/AmazonEKSServicePolicy > /dev/null
-```
-
-Now lets retrieve the **ARN** of this role and store it in a variable. We later
-assign this to the EKS.
+In order to utilize these pre-existing resources, set environment variables that
+can be used when creating resources necessary to deploy Wordpress.
 
 ```bash
-EKS_ROLE_ARN=$(aws iam get-role --output json --role-name "${EKS_ROLE_NAME}" | jq -r .Role.Arn)
+export CLUSTER_NAME=yourclustername
+export REGION=yourclusterregion
+export VPC_ID=yourvpcid
+export SUBNET_ONE_ID=yourpublicsubnetoneid
+export SUBNET_TWO_ID=yourpublicsubnettwoid
+export SUBNET_THREE_ID=yourpublicsubnetthreeid
 ```
 
-#### Creating the EKS cluster
-
-At this step we create an EKS cluster:
-
-```bash
-CLUSTER_NAME=crossplane-example-cluster
-K8S_VERSION=1.14
-
-aws eks create-cluster \
-  --region "${REGION}" \
-  --name "${CLUSTER_NAME}" \
-  --kubernetes-version "${K8S_VERSION}" \
-  --role-arn "${EKS_ROLE_ARN}" \
-  --resources-vpc-config subnetIds="${SUBNET_IDS}",securityGroupIds="${EKS_SECURITY_GROUP}"
-```
-
-An EKS cluster should have started provisioning, which could take take up to 15
-minutes. You can check the status of the cluster by periodically running:
-
-```bash
-aws eks describe-cluster --name "${CLUSTER_NAME}" --region "${REGION}" | jq -r .cluster.status
-```
-
-Once the provisioning is completed, the above command will return `ACTIVE`.
-
 #### Configuring `kubectl` to communicate with the EKS cluster
 
-Once the cluster is created and is `ACTIVE`, we configure `kubectl` to target
-this cluster:
+In order to communicate with with your EKS cluster, you must configure
+`kubectl`.
```

Comment:
Given that we're (thankfully) removing the preface in which we teach folks how to setup EKS, do we still feel like it's useful to teach folks how to setup `kubectl`? I suggest we remove this and include a link to the EKS docs.